### ISSUE TITLE
pretty print() method would be nice

### ISSUE THREAD

**Issue by apsteinmetz** [tidyverse/duckplyr#177]
There is no explicit print method for class `duckplyr_df`.  Something prettier than the base R method would be nice.  I use this:
```
print.duckplyr_df <- function(x,...){print(class(x));print(as_tibble(x),...)}
```

**Comment by krlmlr** [comment]
Thanks, Art. A tibble converted to a duckframe will still print as a tibble:

``` r
data.frame(a = 1) |>
  duckplyr::as_duckplyr_df()
#>   a
#> 1 1
```

``` r

tibble::tibble(a = 1) |>
  duckplyr::as_duckplyr_df()
#> # A tibble: 1 × 1
#>       a
#>   <dbl>
#> 1     1
```

<sup>Created on 2024-06-08 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

Does this help?

**Comment by hadley** [comment]
To me, it feels like creating a tibble should be the default.

**Comment by krlmlr** [comment]
Should we do

```r
as_duckplyr_df <- function(..., class = class(tibble()))
```

with an option to override?

**Comment by hadley** [comment]
Yeah, I think so.

**Comment by krlmlr** [comment]
Done in bc7c8de7ecd7264ca4c5be93223423365943c136 for `df_from_file()` . Not yet sure about `as_duckplyr_df()`, it's used in 888 places now, and likely to break.

Maybe `as_duckplyr_tibble()` and `as_duckplyr_tbl()` ?

**Comment by hadley** [comment]
I think `as_duckplyr_tibble()` should be adequate.

**Comment by krlmlr** [comment]
Available in duckplyr 0.4.1.
